src/App.js

```diff
@@ -44,21 +46,32 @@ export default class App extends React.Component {
 		}
 	};
 	toggleTheme = () => {
-		let theme = !this.state.on;
+		const theme = !this.state.on;
 		this.setState({ on: theme });
 		this.setInStorage('theme', theme);
```

Comment:
here, you might want to stringify this, because you will save the string true\false in storage